book/website/collaboration/academic-industry/case-studies-and-best-practices/intro-case-studies-and-best-practices-in-academic-industry-collaboration.md

```diff
@@ -0,0 +1,21 @@
+# Introduction
+The creation of this subchapter was motivated by a need to address the realities of navigating the complex relationship between Open Science and Industry, particularly in cases where research is partly or fully funded by industry. A balance must be struck between producing research that meets the priorities of the sponsoring organisation and producing research that is open, shareable, and beneficial to the community. 
+
+Researchers and practitioners championing Open Science could benefit from case studies and lessons learnt from others who have successfully and/or unsuccessfully navigated this aspect of Academic-Industry collaborations.
+
+This subchapter is a continuously growing resource library of case studies in industry-academia collaboration, as well as a collection of existing resources on industry-academia best practices.
+
+## Who this subchapter is for
+
+This subchapter could be especially useful for anyone who is conducting or has conducted research in the context of academic-industry collaboration. This could include:
+- Academics who have an opportunity to work with an industrial partner and are looking for guidance on how to best navigate the relationship to ensure a productive and open collaboration.
```

Comment:
Are we only aiming at people who already have a connection or could the scoping of collaborations also fall under this chapter?
eg
- Academics who's research is suited for academic-industry collaboration who want to see examples of how to identify, approach and build trust with a potential collaboration partner.

Reply:
Personally I think this would be within scope!